kerl

```diff
@@ -411,8 +411,10 @@ assert_perl()
 
 get_javac_version()
 {
-    java_loc=$(which javac)
-    if [ -z "$java_loc" ]; then
+    # http://apple.stackexchange.com/a/135079
+    /usr/libexec/java_home >/dev/null 2>&1
```

Comment:
Does that file not exist when you install OpenJDK on a Linux box? Hmm. Maybe for Linux we just directly check to see if javac exists...